tasks.py

```diff
@@ -0,0 +1,117 @@
+import os
+from pathlib import Path
+from shutil import which
+
+from invoke import task
+from invoke.util import cd
+
+TOOLS = ["poetry", "pre-commit"]
+PORT = os.environ.get("SERVER_PORT", 8000)
+
+ACTIVE_VENV = os.environ.get("VIRTUAL_ENV", None)
+VENV_HOME = Path(os.environ.get("WORKON_HOME", "~/.local/share/virtualenvs"))
+VENV_PATH = Path(ACTIVE_VENV) if ACTIVE_VENV else (VENV_HOME / "kagi")
+VENV = str(VENV_PATH.expanduser())
+
+PRECOMMIT = which("pre-commit") if which("pre-commit") else (VENV / "pre-commit")
+POETRY = which("poetry") if which("poetry") else (VENV / "poetry")
+
+
+@task
+def build(c, docs=True):
+    """Build documentation"""
+    if docs:
+        c.run(f"{VENV}/bin/sphinx-build docs docs/_build")
+
+
+@task(build)
+def livedocs(c):
+    """Serve docs at http://localhost:$PORT/ (default port is 8000)"""
+    from livereload import Server
+
+    server = Server()
+    server.watch("docs/conf.py", lambda: build(c))
+    server.watch("CONTRIBUTING.rst", lambda: build(c))
+    server.watch("docs/*.rst", lambda: build(c))
+    server.serve(port=PORT, root="docs/_build")
+
+
+@task
+def serve(c):
+    """Serve site at https://localhost:$PORT/ (default port is 8000)"""
+    with cd("testproj"):
+        c.run(
+            f"{VENV}/bin/python manage.py runserver_plus "
```

Comment:
I realized while writting the troubleshooting, that everything works fine in localhost even if we don't use SSL. I guess would make it easier for people if we relied directly on `runserver`

Reply:
Agreed. See #29.